suites/test_assembly_content.py

```diff
@@ -104,19 +105,18 @@ def verify_assembly_content(self, api_auth, setup_test_products):
           check_that("Modules included",
                      data_from_published_assembly.json()["assembly"]["modules_included"][i],
                      all_of(has_entry("canonical_uuid"), has_entry("level_offset"), has_entry("module_uuid"),
-                            has_entry("title"), has_entry("url"), has_entry("pantheon_env"), has_entry("relative_url")))
-          check_that("Modules included-> relative_url", published_module_url, contains_string(
-              data_from_published_assembly.json()["assembly"]["modules_included"][i]["relative_url"]))
+                            has_entry("title"), has_entry("url"), has_entry("pantheon_env"),
+                            has_entry("relative_url", equal_to("/"+published_module_relative_url))))
```

Comment:
@noopurmath3 this check is likely to fail when the included modules is 2, and it iterates over but checks for "equal to "published_module_relative_url" as that has not been extracted for every module. Please add the correctness of the checks.
```
 CHECK | Expect Number of Modules included to be greater than or equal to 1                                                                                                  | Got 2                                                  |
+-------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------+
| CHECK | Expect Modules included :                                                                                                                                           | Got:                                                   |
|       |     - to have entry "canonical_uuid"                                                                                                                                |     - to have entry "canonical_uuid" => OK, got        |
|       |     - and to have entry "level_offset"                                                                                                                              | "bb344853-d3b0-46e5-9ed9-17eb3b43aa15"                 |
|       |     - and to have entry "module_uuid"                                                                                                                               |     - and to have entry "level_offset" => OK, got "1"  |
|       |     - and to have entry "title"                                                                                                                                     |     - and to have entry "module_uuid" => OK, got       |
|       |     - and to have entry "url"                                                                                                                                       | "b35b4d18-a0a3-443c-97f3-5ae7f2b003b8"                 |
|       |     - and to have entry "pantheon_env"                                                                                                                              |     - and to have entry "title" => OK, got "Module     |
|       |     - and to have entry "relative_url" that is equal to "/api/module/variant.json/69deea2d-5168-4a15-9064-8dbfe06a08b0"                                             | api-tests-ci-2 uploader | Activating or deactivating   |
|       |                                                                                                                                                                     | system services in the web console"                    |
|       |                                                                                                                                                                     |     - and to have entry "url" => OK, got ""            |
|       |                                                                                                                                                                     |     - and to have entry "pantheon_env" => OK, got "qa" |
|       |                                                                                                                                                                     |     - and to have entry "relative_url" that is equal   |
|       |                                                                                                                                                                     | to "/api/module/variant.json/69deea2d-5168-4a15-9064-8 |
|       |                                                                                                                                                                     | dbfe06a08b0" => KO, got ""                             |
+-------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------+
| CHECK | Expect Modules included-> pantheon_env to be equal to "qa"                                                                                                          | Got "qa"                                               |
+-------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------+
| CHECK | Expect Modules included :                                                                                                                                           | Got:                                                   |
|       |     - to have entry "canonical_uuid"                                                                                                                                |     - to have entry "canonical_uuid" => OK, got        |
|       |     - and to have entry "level_offset"                                                                                                                              | "907f7a5e-ee80-4034-b1d6-be3a20fc0c7b"                 |
|       |     - and to have entry "module_uuid"                                                                                                                               |     - and to have entry "level_offset" => OK, got "0"  |
|       |     - and to have entry "title"                                                                                                                                     |     - and to have entry "module_uuid" => OK, got       |
|       |     - and to have entry "url"                                                                                                                                       | "40f9130c-b146-47c0-99d5-f3b2d3fb0557"                 |
|       |     - and to have entry "pantheon_env"                                                                                                                              |     - and to have entry "title" => OK, got "Content    |
|       |     - and to have entry "relative_url" that is equal to "/api/module/variant.json/69deea2d-5168-4a15-9064-8dbfe06a08b0"                                             | Test Module | Logging in to the web console using      |
|       |                                                                                                                                                                     | Kerberos authentication (Image present)"               |
|       |                                                                                                                                                                     |     - and to have entry "url" => OK, got               |
|       |                                                                                                                                                                     | "https://pantheon2-qa.int.us-east.aws.preprod.paas.red |
|       |                                                                                                                                                                     | hat.com/api/module/variant.json/907f7a5e-ee80-4034-b1d |
|       |                                                                                                                                                                     | 6-be3a20fc0c7b"                                        |
|       |                                                                                                                                                                     |     - and to have entry "pantheon_env" => OK, got "qa" |
|       |                                                                                                                                                                     |     - and to have entry "relative_url" that is equal   |
|       |                                                                                                                                                                     | to "/api/module/variant.json/69deea2d-5168-4a15-9064-8 |
|       |                                                                                                                                                                     | dbfe06a08b0" => KO, got "/api/module/variant.json/907f |
|       |                                                                                                                                                                     | 7a5e-ee80-4034-b1d6-be3a20fc0c7b"                      |
+-------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------+
| CHECK | Expect Modules included-> pantheon_env to be equal to "qa"                                                                                                          | Got "qa"                                               |
```

Reply:
updated the logic for verifying relative_url